EasyPost/Client.cs

```diff
@@ -6,59 +6,58 @@ namespace EasyPost
 {
     public class Client : EasyPostClient
     {
-        public AddressService Address => GetService<AddressService>();
+        public AddressService Address { get; }
 
-        public ApiKeyService ApiKey => GetService<ApiKeyService>();
+        public ApiKeyService ApiKey { get; } // TODO: Recommend renaming. Because ApiKey is used as the name of the API key service, you have to access the actual configured API key via ApiKeyInUse
 
-        public BatchService Batch => GetService<BatchService>();
+        public BatchService Batch { get; }
 
         // ReSharper disable once MemberCanBePrivate.Global
         // ReSharper disable once UnusedAutoPropertyAccessor.Global
         public BetaClient Beta { get; }
```

Comment:
Question: I believe C# is the only thread-safe CL that has a `Beta` namespace, and the rest of thread-safe CLs just prepend `Beta` in the class service name. Should we make this change in the V5 major bump?

Reply:
I am leaning toward making this change in order to maintain consistency with other CLs.